[2025-03-12T10:08:00.097+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-12T10:08:00.137+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weekly_category_analysis.orders manual__2025-03-12T10:07:47.232032+00:00 [queued]>
[2025-03-12T10:08:00.155+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weekly_category_analysis.orders manual__2025-03-12T10:07:47.232032+00:00 [queued]>
[2025-03-12T10:08:00.156+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-12T10:08:00.182+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): orders> on 2025-03-12 10:07:47.232032+00:00
[2025-03-12T10:08:00.191+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=567) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-12T10:08:00.194+0000] {standard_task_runner.py:72} INFO - Started process 573 to run task
[2025-03-12T10:08:00.195+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'weekly_category_analysis', 'orders', 'manual__2025-03-12T10:07:47.232032+00:00', '--job-id', '456', '--raw', '--subdir', 'DAGS_FOLDER/s3_data.py', '--cfg-path', '/tmp/tmpca2mjz_j']
[2025-03-12T10:08:00.197+0000] {standard_task_runner.py:105} INFO - Job 456: Subtask orders
[2025-03-12T10:08:00.276+0000] {task_command.py:467} INFO - Running <TaskInstance: weekly_category_analysis.orders manual__2025-03-12T10:07:47.232032+00:00 [running]> on host b404e9833d11
[2025-03-12T10:08:00.436+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weekly_category_analysis' AIRFLOW_CTX_TASK_ID='orders' AIRFLOW_CTX_EXECUTION_DATE='2025-03-12T10:07:47.232032+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-12T10:07:47.232032+00:00'
[2025-03-12T10:08:00.439+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-03-12T10:08:00.440+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-03-12T10:08:00.441+0000] {logging_mixin.py:190} INFO - Current task name:orders state:running start_date:2025-03-12 10:08:00.138913+00:00
[2025-03-12T10:08:00.442+0000] {logging_mixin.py:190} INFO - Dag name:weekly_category_analysis and current dag run status:running
[2025-03-12T10:08:00.442+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-12T10:08:01.054+0000] {httpchecksum.py:481} INFO - Skipping checksum validation. Response did not contain one of the following algorithms: ['crc32', 'sha1', 'sha256'].
[2025-03-12T10:08:01.215+0000] {logging_mixin.py:190} INFO - Validation successful. Found 1615 order records.
[2025-03-12T10:08:01.216+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-12T10:08:01.231+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-12T10:08:01.232+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=weekly_category_analysis, task_id=orders, run_id=manual__2025-03-12T10:07:47.232032+00:00, execution_date=20250312T100747, start_date=20250312T100800, end_date=20250312T100801
[2025-03-12T10:08:01.271+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-03-12T10:08:01.271+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-03-12T10:08:01.272+0000] {logging_mixin.py:190} INFO - Dag name:weekly_category_analysis queued_at:2025-03-12 10:07:47.246755+00:00
[2025-03-12T10:08:01.274+0000] {logging_mixin.py:190} INFO - Task hostname:b404e9833d11 operator:PythonOperator
[2025-03-12T10:08:01.293+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-12T10:08:01.367+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-12T10:08:01.370+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
